model/HTMLValue.php

```diff
@@ -55,6 +54,11 @@ public function getContent() {
 			return Convert::raw2att($attrs[$matches[0]]);
 		}, $res);
 
+		// Prevent &nbsp; being encoded as literal utf-8 characters
```

Comment:
It was an issue we never bothered to fix before. It was affecting change detection, because tinymce was quite rightly encoding the UTF-8 equivalent back to `&nbsp;`, and causing change detection to trigger the "you have unsaved changes" warning even when nothing was touched by the user.

It was easier to make the core follow the standard than to hack change detection.